x-pack/legacy/plugins/code/server/lsp/go_launcher.ts

```diff
@@ -104,17 +102,18 @@ export class GoServerLauncher extends AbstractLauncher {
     const goCache = path.resolve(goPath, '.cache');
     const params: string[] = ['-port=' + port.toString()];
     const golsp = path.resolve(installationPath, launchersFound[0]);
+    const env = process.env;
+    env.PATH = process.platform === 'win32' ? goHome + ';' + env.PATH : goHome + ':' + env.PATH;
```

Comment:
use code like this:

https://github.com/elastic/javascript-typescript-langserver/blob/master/src/dependency-manager.ts#L37

Reply:
done.